presidio-analyzer/tests/test_batch_analyzer_engine.py

```diff
@@ -34,7 +34,7 @@ def test_analyze_iterator_returns_list_of_recognizer_results(
     texts, expected_output, batch_analyzer_engine_simple
 ):
 
-    results = batch_analyzer_engine_simple.analyze_iterator(texts=texts, language="en")
+    results = batch_analyzer_engine_simple.analyze_iterator(texts=texts, language="en", batch_size=2)
```

Comment:
I think this test needs to be also as:
batch_analyzer_engine_simple.analyze_iterator(texts=texts, language="en")
To make sure the functionality is working properly without the batch size.